### PR TITLE
Feature/questionnaire fix

### PR DESCRIPTION
**Pull request by Yinci**
Check whether we need to add extra logic to the questionnaire page (e.g. adding new fields manually)

### CODE REVIEW

**Comment by Yinci** [comment]
This PR makes the following changes:

- All `@lang` translations have been moved from `woningdossier.php` to its own lang file
- Validation has been added to check whether question options are empty or not; this is required to stop exceptions happening when adding questions with options (`index options is undefined`)
- QoL changes to the editor; 
  - Add-validation button disappears when clicked, so no more than one can be added (only 1 is currently supported)
  - Validation can be removed
  - An add-option button has been added to questions that can have options, to manually allow the user to add new options
  - Index logic updates when deleting an option
  - If validation fails, previously added questions, options, validation, etc. get rebuild
  - If validation fails, the inputs on which it failed will now be highlighted
  - Validation errors are now properly translated
- Split off the JS into a separate part
- Javascript refactor, to be more consistent, and allow for easier maintenance
- Javascript now uses `@lang` as well.
- LanguageRequired rule to allow easy checks of locale input
- Removed unused parts

**Comment by Yinci** [comment]
- Added a test to test for a question with an option

File: composer.lock [hunk 1]
```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "a036552384555416f7891027b2f3cf7c",
+    "content-hash": "766b1653d7c8aa13cecf2fda0dcbec15",
```
Comment (frogeyedman) [inline composer.lock:7]: Composer update should be reverted

**Review by frogeyedman — CHANGES_REQUESTED** [review]

**Review by frogeyedman — CHANGES_REQUESTED** [review]
A radio button question does not give a validation error when its supposed to.

- Add a new radio button question
- Leave the question name empty, but fill in the options
- Save the questionnaire

**Review by frogeyedman — APPROVED** [review]
Seems ok.

**Comment by frogeyedman** [comment]
@Yinci could you solve the conflicts and or check if still relevant ?

**Comment by Yinci** [comment]
Potentially rewrite this using Livewire and AlpineJS now develop has been upgraded. Definitely requires checking if still functional if not going to be rewritten.

**Comment by frogeyedman** [comment]
Closing PR because author states upgrade with alpine js / livewire is better, which i support. Also has a bug shown on the test environment.

**Pull request closed without merging**